docs/guides/dapis/index.md

```diff
@@ -127,19 +127,31 @@ dashboard through the [API3 Market](https://market.api3.org/).
 To read more about reading a dAPI value,
 [click here](/guides/dapis/read-a-dapi/).
 
-## How many providers are there for dAPIs?
+## Data providers for managed dAPIs
 
-Currently, there are 7 providers for Managed dAPIs:
+There are nine providers used by managed dAPIs as a data source.
 
-| Asset Type     | Data Provider                                                           |
-| -------------- | ----------------------------------------------------------------------- |
-| Cryptocurrency | Nodary, Coinpaprika, Finage, Twelvedata, NCFX, Kaiko, dxFeed            |
-| Forex          | Nodary, Finage, Twelvedata, NCFX, IEXCloud, Finnhub, dxFeed, Tradermade |
-| Commodities    | Nodary, Finage, Twelvedata, dxFeed, Tradermade                          |
-| Equities       | Nodary, Finage, Twelvedata, IEXCloud, Finnhub, dxFeed                   |
+- Coinpaprika
+- dxFeed
+- Finage
+- Finnhub
+- IEXCloud
+- Kaiko
+- NCFX
+- Tradermade
+- Twelvedata
 
```

Comment:
Maybe a transition sentence between the list and table? For example:

These providers can be grouped by asset type(s) of supplied data: